package.json

```diff
@@ -9,7 +9,8 @@
   ],
   "author": "Thuzi LLC <pshrestha@thuzi.com> (https://github.com/Thuzi)",
   "contributors": [
-    "Daniel Friesen <d@danf.ca> (http://danf.ca)"
+    "Daniel Friesen <d@danf.ca> (http://danf.ca)",
+    "Shibu Lijack <shibulijack@gmail.com> (https://github.com/shibulijack"
```

Comment:
Missing `)`, if you're going to make yourself the only one of the many people who has submitted PRs to this library attributed here.

Reply:
Sorry but I don't understand. I'll fix the missing parenthesis but am I missing anything else here?

Reply:
No, I was just noting that there are [16 other](https://github.com/node-facebook/facebook-node-sdk/graphs/contributors) people who have contributed code to this library and none of them are credited in the contributors list. It's really only listing me because I took over as maintainer.

Reply:
Alright. I'll remove myself from the contributor list. 
Just curious though, shouldn't all the 16 people be included in the contributor list?

Reply:
Perhaps, but in general I think most npm packages bother adding everyone to the contributors list. I sometimes see a CONTRIBUTORS file. But the contributors list is usually kept short, presumably because it's downloaded by everyone that installs the library.